ci/Jenkinsfile

```diff
@@ -112,20 +112,15 @@ timeout(90) {
 
         stage('Upload apk for e2e tests') {
           if (env.CHANGE_ID != null) {
-
-              withCredentials([
-                  string(credentialsId: 'SAUCE_ACCESS_KEY', variable: 'key'),
-                  string(credentialsId: 'SAUCE_USERNAME', variable: 'username')
-              ]) {
-                def apk_name = env.CHANGE_ID + '.apk'
-                sh('curl -u ' + username+ ':' + key + ' -X POST -H "Content-Type: application/octet-stream" https://saucelabs.com/rest/v1/storage/' + username + '/' + apk_name + '?overwrite=true --data-binary @android/app/build/outputs/apk/release/app-release.apk')
-              }
-
+              env.SAUCE_LABS_APK = 'im.status.ethereum-e2e-' + env.CHANGE_ID + '.apk'
               withCredentials([
+                  string(credentialsId: 'SAUCE_ACCESS_KEY', variable: 'SAUCE_ACCESS_KEY'),
+                  string(credentialsId: 'SAUCE_USERNAME', variable: 'SAUCE_USERNAME'),
                   string(credentialsId: 'diawi-token', variable: 'DIAWI_TOKEN'),
                   string(credentialsId: 'GIT_HUB_TOKEN', variable: 'GITHUB_TOKEN'),
                   string(credentialsId: 'SLACK_JENKINS_WEBHOOK', variable: 'SLACK_URL')
               ]) {
+                sh 'fastlane android saucelabs'
```

Comment:
I think we should do `bundle exec fastlane android saucelabs` to avoid warnings like this:
```
+ fastlane ios nightly
[15:35:49]: [33mfastlane detected a Gemfile in the current directory[0m
[15:35:49]: [33mhowever it seems like you don't use `bundle exec`[0m
[15:35:49]: [33mto launch fastlane faster, please use[0m
[15:35:49]: 
[15:35:49]: [36m$ bundle exec fastlane ios nightly[0m
[15:35:49]: 
[15:35:49]: [33mGet started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile[0m
```

Reply:
yeah, but I hope to make a followup PR for that, later on